docs/src/main/sphinx/release/release-403.md

```diff
@@ -0,0 +1,38 @@
+# Release 403 (9 Nov 2022)
+
+## General
+
+* Round `TimeWithTimeZone` values when `PARAMETRIC_DATETIME` is not set. ({issue}`14950`)
+* Fix query deadlock in multi-join queries where broadcast join size is
+  underestimated. ({issue}`14948`)
+* Fix incorrect results when `min(x, n)` or `max(x, n)` is used as a window
+  function. ({issue}`14886`)
+* Fix failure for certain queries involving joins over partitioned tables. ({issue}`14317`)
+* Fix incorrect order of parameters in `DESCRIBE INPUT` when they appear in a
+  `WITH` clause. ({issue}`14738 `)
+
+## BigQuery connector
+
+* Improve performance of `SHOW SCHEMAS` by adding a dataset metadata cache. This
```

Comment:
What's a "dataset"? That word seems redundant. I think just saying "adding a metadata cache" is sufficient.